### PR TITLE
News/2021 07 28

### PR DESCRIPTION
**Pull request by dev-yakuza**
See: https://github.com/sider/runners/releases/tag/0.53.2

### CODE REVIEW

**Comment by dev-yakuza** [comment]
- News: https://deploy-preview-656--sider-docs.netlify.app/news/2021
- Secret Scan: https://deploy-preview-656--sider-docs.netlify.app/tools/others/secret-scan

File: docs/news/2021.md [hunk 1]
```diff
@@ -10,6 +10,13 @@ This document describes notable changes on Sider in the year 2021.
 
 ---
 
+## Secret Scan out of beta and enabled by default
+
+<time className="NewsDate" dateTime="2019-11-29">July 28, 2021</time>
```
Comment (ybiquitous) [inline docs/news/2021.md:15]: Ooops 😓 

```suggestion
<time className="NewsDate" dateTime="2021-07-28">July 28, 2021</time>
```

File: docs/news/2021.md [hunk 2]
```diff
@@ -10,6 +10,13 @@ This document describes notable changes on Sider in the year 2021.
...
+
+[Secret Scan](../tools/others/secret-scan.md), which was launched in beta on June,
```
Comment (ybiquitous) [inline docs/news/2021.md:17]: How about adding a link as below?

```suggestion
[Secret Scan](../tools/others/secret-scan.md), which was [launched in beta on June](#secret-scanning-beta),
```

File: docs/news/2021.md [hunk 3]
```diff
@@ -10,6 +10,13 @@ This document describes notable changes on Sider in the year 2021.
 
 ---
 
+## Secret Scan out of beta and enabled by default
+
+<time className="NewsDate" dateTime="2021-07-28">July 28, 2021</time>
+
+[Secret Scan](../tools/others/secret-scan.md), which was [launched in beta on June](#secret-scanning-beta),
+is now out of beta and enabled by default for new repositories.
+
```
Comment (ybiquitous) [inline docs/news/2021.md:19]: [suggestion] How about making it more emotional?

For example:

```
Today, we’ve made Secret Scan (launched in beta in June) out of beta and enabled by default for all repositories (even the existing repositories)! ✨

If you have any questions, please feel free to ask us. 💬
```

File: docs/news/2021.md [hunk 4]
```diff
@@ -10,6 +10,13 @@ This document describes notable changes on Sider in the year 2021.
 
 ---
 
+## Secret Scan out of beta and enabled by default
+
+<time className="NewsDate" dateTime="2021-07-28">July 28, 2021</time>
+
+[Secret Scan](../tools/others/secret-scan.md), which was [launched in beta on June](#secret-scanning-beta),
```
Comment (ybiquitous) [inline docs/news/2021.md:17]: Is `in` grammatically correct?

```suggestion
[Secret Scan](../tools/others/secret-scan.md), which was [launched in beta in June](#secret-scanning-beta),
```

Comment (dev-yakuza) [reply docs/news/2021.md]: @ybiquitous 
Fixed reviews on https://github.com/sider/sider-docs/pull/656/commits/541a1b04219ea5b17eddabe14f162bd859cfd5f3
Thanks!

File: docs/news/2021.md [hunk 5]
```diff
@@ -10,6 +10,14 @@ This document describes notable changes on Sider in the year 2021.
 
 ---
 
+## Secret Scan out of beta and enabled by default
+
+<time className="NewsDate" dateTime="2021-07-28">July 28, 2021</time>
+
+Today, we’ve made [Secret Scan](../tools/others/secret-scan.md) ([launched in beta on June](#secret-scanning-beta)) out of beta and enabled by default for all repositories (even the existing repositories)! ✨
```
Comment (ybiquitous) [inline docs/news/2021.md:17]: How about `on` -> `in`?

```suggestion
Today, we’ve made [Secret Scan](../tools/others/secret-scan.md) ([launched in beta in June](#secret-scanning-beta)) out of beta and enabled by default for all repositories (even the existing repositories)! ✨
```

**Review by ybiquitous — COMMENTED** [review]
It seems good to add a link to the blog:
https://siderlabs.com/blog/announcement-sider-secret-scan/

**Comment by dev-yakuza** [comment]
@ybiquitous 
How about this change(https://github.com/sider/sider-docs/pull/656/commits/62c9172c239fcf7aee4fa34e137d6816448cbe46)?

**Review by ybiquitous — APPROVED** [review]
Nice! LGTM 👍🏼

**Pull request merged**